test/functional/shared/resources/application_test.go

```diff
@@ -54,3 +59,169 @@ func Test_Application(t *testing.T) {
 	})
 	test.Test(t)
 }
+
+func Test_ApplicationGraph(t *testing.T) {
+	// Deploy a simple app
+	template := "testdata/corerp-resources-application-graph.bicep"
+	name := "corerp-application-simple1"
+	appNamespace := "default-corerp-application-simple1"
+
+	frontCntrID := "/planes/radius/local/resourcegroups/kind-radius/providers/Applications.Core/containers/http-front-ctnr-simple1"
+	frontCntrName := "http-front-ctnr-simple1"
+	frontCntrType := "Applications.Core/containers"
+
+	backCntrID := "/planes/radius/local/resourcegroups/kind-radius/providers/Applications.Core/containers/http-back-ctnr-simple1"
+	backCntrName := "http-back-ctnr-simple1"
+	backCntrType := "Applications.Core/containers"
+
+	backRteID := "/planes/radius/local/resourcegroups/kind-radius/providers/Applications.Core/httpRoutes/http-back-rte-simple1"
+	backRteName := "http-back-rte-simple1"
+	backRteType := "Applications.Core/httpRoutes"
+
+	ProvisioningStateSuccess := "Succeeded"
+	DirectionInbound := v20231001preview.DirectionInbound
+	DirectionOutbound := v20231001preview.DirectionOutbound
+
+	expectedGraphResp := v20231001preview.ApplicationsClientGetGraphResponse{
+		ApplicationGraphResponse: v20231001preview.ApplicationGraphResponse{
+			Resources: []*v20231001preview.ApplicationGraphResource{
+				{
+					ID:                &frontCntrID,
+					Name:              &frontCntrName,
+					Type:              &frontCntrType,
+					ProvisioningState: &ProvisioningStateSuccess,
+					OutputResources:   []*v20231001preview.ApplicationGraphOutputResource{},
+					Connections: []*v20231001preview.ApplicationGraphConnection{
+						{
+							Direction: &DirectionInbound,
+							ID:        &backRteID,
+						},
+					},
+				},
+				{
+					ID:                &backCntrID,
+					Name:              &backCntrName,
+					Type:              &backCntrType,
+					ProvisioningState: &ProvisioningStateSuccess,
+					OutputResources:   []*v20231001preview.ApplicationGraphOutputResource{},
```

Comment:
Why are the output resources empty?

Reply:
I just focused the test to check validity of resources and their connections, and not check if outputresources are correct ( since that is not directly related)..